src/main/java/uk/nhs/cdss/audit/model/AuditEntry.java

```diff
@@ -0,0 +1,29 @@
+package uk.nhs.cdss.audit.model;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import java.time.Instant;
+
+/**
+ * Represents calls made by this web service's FHIR client during an audit session
```

Comment:
Think the comments on this and the session object don't hold true on this service as they're more used for storing here than actual capturing of calls.

Reply:
Fair point, I've updated the comments.